config/docker/chromeos-flash/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN echo "nameserver 1.1.1.1" > /etc/resolv.conf && \
+    apt-get update && \
+    apt-get install --no-install-recommends -y \
+    wget\
+    iputils-ping \
+    net-tools \
+    nmap \
+    ssh
+
+RUN mkdir /keys
+
+ADD id_rsa_chromeos_flash /keys/id_rsa_chromeos_flash
```

Comment:
Please add a missing line return to clear the git warning.

Reply:
done